models/user.server.model.js

```diff
@@ -59,6 +61,20 @@ searchUserById = function(id, done) {
 }
 
 /**
+ * Delete an existing forum user matching a given ID
+ * @param id the ID for matching to the database document being deleted
+ * @param done function callback, returns status code and message if error
+ */
+deleteUserById = function(id, done) {
+    User.deleteOne({ _id: id })
+        .then((res) => {
+            if (res.deletedCount === 0) {
+                return done({ err: "Not found", status: 404 });
+            }
+            return done(res);
+        });
+}
+
  * Updates given fields of a database collection document for a user matching a given ID
```

Comment:
```suggestion
/**
 * Updates given fields of a database collection document for a user matching a given ID
```
Sorry, I think I may have caused this bug when resolving the merge conflict in the pull request